server/app.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
+global.__base = __dirname;

```

Comment:
In addition, even if you would set the right folder, if 2 dependencies would set the same __dirname to its dependency and then make it global, then they would conflict each other...